src/controllers/multi-tenancy/MultiTenancyController.ts

```
@@ -1913,4 +1917,124 @@
+  }
+
+  @Security('apiKey')
+  @Post('/credential/verify/:tenantId')
+  public async verifyCredential(
+    @Path('tenantId') tenantId: string,
+    @Body() credentialToVerify: SafeW3cJsonLdVerifyCredentialOptions | any
+  ) {
+    let formattedCredential
+    try {
+      await this.agent.modules.tenants.withTenantAgent({ tenantId }, async (tenantAgent) => {
+        const {credential,  ...credentialOptions}= credentialToVerify
```

ESLint

[error] 'credential' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)